src/test/resources/MissedNotificationOutput.json

```diff
@@ -1 +1 @@
-["{"subscriptionName" : "Subscription_1","AggregatedObject" : {"fileInformation":[{"extension":"jar","classifier":""}],"buildCommand":null,"testCaseExecutions":[{"testCaseFinishEventId":"11109351-41e0-474a-bc1c-f6e81e58a1c9","testCaseStartedTime":1481875925916,"testCaseStartedEventId":"cb9d64b0-a6e9-4419-8b5d-a650c27c59ca","testCaseFinishedTime":1481875935919,"testCase":{"conclusion":"SUCCESSFUL","verdict":"PASSED","tracker":"My Other Test Management System","id":"TC5","uri":"https://other-tm.company.com/testCase/TC5"}}],"id":"6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43","time":1481875891763,"type":"ARTIFACT_1","gav":{"groupId":"com.mycompany.myproduct","artifactId":"sub-system","version":"1.1.0"}}}"]
\ No newline at end of file
+["{"subscriptionName": "Subscription_1","AggregatedObject": {"fileInformation": [{"extension": "jar","classifier": ""}],"buildCommand": null,"testCaseExecutions": [{"testCaseFinishEventId": "11109351-41e0-474a-bc1c-f6e81e58a1c9","testCaseStartedTime": 1481875925916,"testCaseStartedEventId": "cb9d64b0-a6e9-4419-8b5d-a650c27c59ca","testCaseFinishedTime": 1481875935919,"testCase": {"conclusion": "SUCCESSFUL","verdict": "PASSED","tracker": "My Other Test Management System","id": "TC5","uri": "https://other-tm.company.com/testCase/TC5"}}],"id": "6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43","time": 1481875891763,"type": "ARTIFACT_1","gav": {"groupId": "com.mycompany.myproduct","artifactId": "sub-system","version": "1.1.0"}}}"]
```

Comment:
What is this file used for? Is it output generated file from EI?

Reply:
This is response of /queryMissedNotifications/{subscriptionName} call. It is used for test.

Reply:
Ok,, so it is used for Expect result in test.

Reply:
Yes